telemetry/definitions/commonDefinitions.json

```diff
@@ -3808,6 +3823,28 @@
                 }
             ]
         },
+        {
+            "name": "codeTransform_jobMavenRunTime",
+            "description": "A timestamp for when all maven commands are done executing",
```

Comment:
A timestamp is not an event. It's a field. Is there an existing event that can capture this? You can add fields to an existing event.

Reply:
Currently codeTransform_jobCreateZipEndTime captures the time of maven commands execution and the time of zip file creation combined. We want to track them separately. The only related existing event is codeTransform_mvnBuildFailed, but it only fires when maven exection fails.

I changed the name to make it more descriptive.

Reply:
> The only related existing event is codeTransform_mvnBuildFailed, but it only fires when maven exection fails.

That's why that metric should be named `codeTransform_mvnBuild`. "Failed" or "success" are stored in the `result` field, it doesn't make sense to have a metric specifically for the "failed" case.